deployment/nginx.default

```diff
@@ -0,0 +1,13 @@
+server {
```

Comment:
Can we add some comments here so that this becomes self explanatory for the developer trying to change the configurations here ?